Test/dafny0/Comprehensions.dfy.expect

```diff
@@ -64,11 +64,12 @@ Execution trace:
     (0,0): anon9_Else
     (0,0): anon10_Then
     (0,0): anon11_Then
-Comprehensions.dfy(189,16): Error: value does not satisfy the subset constraints of 'nat'
+Comprehensions.dfy(189,35): Error: value does not satisfy the subset constraints of 'nat'
```

Comment:
Nice improvement in source-location precision!

Reply:
OK